README.md

```diff
@@ -23,8 +23,8 @@ Or use the version without the editor: [Version without editor](https://raw.gith
 Add the following to resources in your lovelace config:
 
 ```yaml
-- url: https://cdn.jsdelivr.net/gh/bramkragten/weather-card/dist/weather-card.min.js
-  type: module
+  - type: js
```

Comment:
Why type `js`? It is deprecated.

Reply:
Odd, it was not working until I switched to type: js